base/process_utils.h

```diff
@@ -0,0 +1,28 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BASE_PROCESS_UTILS_H_
+#define BRAVE_BASE_PROCESS_UTILS_H_
+
+#include <string>
+
+#include "base/process/launch.h"
+#include "base/process/process.h"
+#include "third_party/abseil-cpp/absl/types/optional.h"
+
+namespace brave {
+class ProcessUtil {
+  ProcessUtil();
+  ~ProcessUtil();
+
+ public:
+  static absl::optional<std::string> ReadAppOutput(base::CommandLine cmdline,
```

Comment:
can you add comments about what is this and why this is needed? also it's not clear why this is not a free function

Reply:
Added a comment. It is a little bit easier to add friend class than friend free function here 
https://github.com/brave/brave-core/pull/17288/files#diff-388a4b1d0f26ffc4aa94bfc430ffe81183e24d9bfd516bec0c53b43af3f611fd